app/index.html

```diff
@@ -377,8 +398,12 @@ <h5 style="margin-top: 1em">Visualization state</h5>
 
                     await app.fetchAndLoad(externalDataUrl(external), config);
                 } else {
-                    // default to Arginine-Dipeptide
-                    await app.loadExample('Arginine-Dipeptide');
+                    // loads a random example
+                    var exampleKeys = Object.keys(examples);
+                    var randomIndex = Math.floor(Math.random() * exampleKeys.length);
+                    var randomExampleKey = exampleKeys[randomIndex];
+
+                    await app.loadExample(randomExampleKey);
```

Comment:
True, but `chemiscope.min.js` is 3MB already so even in the worst case it will make loading 2x slower.... 
Maybe we can test a bit and see if it's a problem, and in case I think the "right thing" would be to reduce a bit the size of the demo datasets.